src/components/calcite-card/calcite-card.tsx

```diff
@@ -0,0 +1,157 @@
+import { Component, Element, Host, Method, Prop, h } from "@stencil/core";
+
+import { getElementDir } from "../../utils/dom";
+import { CSS_UTILITY } from "../../utils/resources";
+import { VNode } from "@stencil/core/dist/declarations";
+
+import { CSS, SLOTS } from "./resources";
+import CalciteScrim from "../../utils/CalciteScrim";
+import classnames from "classnames";
+
+/**
```

Comment:
Add doc for default slot

```
@slot - Default desc.
```

Reply:
we don't have a default slawt. should there be one?

Reply:
@kat10140 @asangma there's a slot for a heading but not a default slot for content? Is content not allowed? Seems odd to me.

In the material card, there's a content paragraph area. 
![image](https://user-images.githubusercontent.com/1231455/69187897-4e382880-0ad0-11ea-9938-32c663b19b6d.png)

Reply:
right, we had the subhead and supporting text props before, but they were removed. @asangma how you wanna do it? Did you want to change the header back to a prop and have a content prop like you were talking about before, or add a default slot?

Reply:
(having header as a prop wouldn't allow for adding an svg like we'd discussed)

Reply:
I just added and doc'd default slot. Alan, let me know if you'd like to change this. Working on tests now